setup.py

```diff
@@ -1,41 +1,38 @@
 import re
-from setuptools import setup
 
+from setuptools import setup
 
 requirements = []
-with open("requirements.txt", "r", encoding="utf-8") as f:
-    requirements = f.readlines()
 
-version = ""
-with open("discord/__init__.py", encoding="utf-8") as f:
-    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]', f.read(), re.MULTILINE).group(1)  # type: ignore
 
-if not version:
-    raise RuntimeError("version is not set")
+VERSION = "2.3.0"
```

Comment:
Is this hard-coded for a reason?

Reply:
Yes, once this works we will be moving it to the main repo where it will be able to use the disnake version.